src/admin.cc

```diff
@@ -155,14 +155,15 @@ rd_kafka_event_t* PollForEvent(
   int timeout_ms) {
   // Establish what attempt we are on
   int attempt = 0;
+  int attempt_timeout_ms = max_tries > 0 ? timeout_ms / max_tries : timeout_ms;
```

Comment:
@webmakersteve 
All calls to this method will make 5 attempts. The original timeout was `1000ms` but these 5 attempts will effectively turn it into `5000ms` timeout.
Here we'll ensure that timeout is fixed no matter how many attempts there are (it's not exactly fixed for example when `timeout_ms / max_tries` division ends up with some non-zero modulo but that's likely irrelevant deviation)
This might be considered as a breaking change because the default timeout was effectively `5000ms` although it was set to `1000ms`. This is why I increased this default value to 5000 in `lib/admin.js`